pySDC/implementations/problem_classes/Burgers.py

```diff
@@ -125,9 +130,12 @@ def plot(self, u, t=None, fig=None, comp='u'):  # pragma: no cover
 
 
 class Burgers2D(GenericSpectralLinear):
+    """
+    See documentation of `Burgers1D`.
+    """
+
     dtype_u = mesh
     dtype_f = imex_mesh
-    xp = np
 
     def __init__(self, nx=64, nz=64, epsilon=0.1, fux=2, fuz=1, mode='T2U', **kwargs):
```

Comment:
why the `**kwargs` here ?

Reply:
Possible keyword arguments are `useGPU`, for instance.

Reply:
Why is it not explicit then ?

Reply:
Because this it is passed to the parent class, so that the individual problems don't need to worry about this.

Reply:
Can spyder not autocomplete keyword arguments for parent classes?

Reply:
Depends ... But I see now : yet another point in favor of writing problem class documentation :smiling_imp: